### PR TITLE
Support negative integers

### PR DESCRIPTION
**Pull request by firoxer**
Hi,

here's a proposed implementation for supporting negative integers.

I noticed that `up_to` wouldn't sound good anymore when it's certain that the input will be negative, so I aliased `up_to` to `up_from` for those cases. This makes the usage look like `fizzbuzz:up_to(123). fizzbuzz:up_from(-123).`. I hope this is fine.

### CODE REVIEW

**Review by tkriik — CHANGES_REQUESTED** [review]
Quality stuff, thanks for the contribution. However, I propose the following change to the module API:

 * Implement a new function `to`, that goes either up or down, depending on the sign of the given number.

Since this new function is more generic, we should deprecate the old `up_to` -function by adding the following line above the function declaration:

`-deprecated({up_to, 1})`

This is to ensure that any projects depending on the old function have ample time to migrate away to the better and more versatile concurrent non-deterministic Fibonacci function, `to`.

**Comment by firoxer** [comment]
Deprecating `up_to/1` is a GOOD idea. However, my Erlang compiler couldn't understand `-deprecated` so I used `%% @deprecated` instead. I hope this is okay.

The rest of the requested changes are provided as requested.

**Comment by tkriik** [comment]
> Deprecating `up_to/1` is a GOOD idea. However, my Erlang compiler couldn't understand `-deprecated` so I used `%% @deprecated` instead. I hope this is okay.

This is not okay at all, since the users of this module are not going to read the source code for deprecation warnings. Add this line under the top `-module(...)` definition:

```
-deprecated({up_to, 1}).
```

which should work, and if not, then fix your Erlang compiler.

**Comment by firoxer** [comment]
Turns out the compiler did not work because I forgot the period

File: fizzbuzz.erl [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
 
+to(N) ->
+    Self = self(),
+    Numbers = if N < 0 -> lists:seq(-1, N, -1); true -> lists:seq(1, N) end,
+    lists:foreach(fun(M) ->
+                          spawn_link(?MODULE, fizzbuzz, [Self, M])
+                  end, Numbers),
+    wait(abs(N)).
+
+%% @deprecated Please use {@link fizzbuzz:to/1} instead.
```
Comment (tkriik) [inline fizzbuzz.erl]:
```suggestion
```

**Review by tkriik — CHANGES_REQUESTED** [review]
One final suggestion, after which this is ready to be merged.

**Review by tkriik — APPROVED** [review]
Looks good to me now. Thank you for contributing to open source!

**Pull request merged**